Meta/CMake/jakt.cmake

```diff
@@ -0,0 +1,39 @@
+#
+# Builds the jakt bootstrap compiler as a host tool for Lagom to compile files written in jakt
+#
+
+include(FetchContent)
+
+FetchContent_Declare(
+  Corrosion
+  GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
+  GIT_TAG v0.2.1
+)
+
+FetchContent_MakeAvailable(Corrosion)
+
+FetchContent_Declare(jakt
+    GIT_REPOSITORY https://github.com/ADKaster/jakt.git
```

Comment:
It's in the SerenityOS org now, pls upstream any changes needed and update this

Reply:
hupdated now that my PR is merged :partying_face: